From f3248c6ef6ca70e28aec2bd7e7e8561b1e06a350 Mon Sep 17 00:00:00 2001
From: Yu Yin <xwafish@gmail.com>
Date: Wed, 29 May 2019 03:03:59 +0000
Subject: [PATCH] [mips64] Corrected grammatical error

see https://crrev.com/c/1630678
that patch modify this by mistake.

Change-Id: I7db0205a08beff3f7e6372d62dd810ef859c9fcc
Reviewed-on: https://chromium-review.googlesource.com/c/v8/v8/+/1633932
Auto-Submit: Yu Yin <xwafish@gmail.com>
Reviewed-by: Yang Guo <yangguo@chromium.org>
Commit-Queue: Yang Guo <yangguo@chromium.org>
Cr-Commit-Position: refs/heads/master@{#61909}
---

diff --git a/deps/v8/src/diagnostics/mips64/disasm-mips64.cc b/deps/v8/src/diagnostics/mips64/disasm-mips64.cc
index ad89083..5edb783 100644
--- a/deps/v8/src/diagnostics/mips64/disasm-mips64.cc
+++ b/deps/v8/src/diagnostics/mips64/disasm-mips64.cc
@@ -744,7 +744,7 @@
       return 4;
     }
     case 'i': {  // 'imm16u or 'imm26.
-      if (format[`] == '1') {
+      if (format[3] == '1') {
         if (format[4] == '6') {
           DCHECK(STRING_STARTS_WITH(format, "imm16"));
           switch (format[5]) {
